File: GestaoHYS/GestaoHYS.API/ClientApp/src/app/_models/PaymentTerms.ts
export class PaymentTerms{
    id: string;
    description: string;
    paymentTermsKey: string;
  
    constructor(id : string, name: string, key: string){
        this.id = id;
        this.description = name;
        this.paymentTermsKey = key;
    }
}

File: GestaoHYS/GestaoHYS.Core/Models/PaymentTerms.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace GestaoHYS.Core.Models
{
    public class PaymentTerms
    {
        [Column("id")]
        [JsonProperty("id")]
        public string id { get; set; }

        [Column("paymentTermsKey")]
        [JsonProperty("paymentTermsKey")]
        public string paymentTermsKey { get; set; }

        [Column("description")]
        [JsonProperty("description")]
        public string description { get; set; }
    }
}
